aerogear-docs.adoc

```diff
@@ -0,0 +1,70 @@
+= Aerogear.org Website Renewal
+
+== Summary
+Everyone should be able to contribute to aerogear.org docs with ease and effectiveness, regardless of the scope of their proposed change.
+At the moment, changes are difficult to author and manage, and we hope to extend the site to include mobile.next and feedhenry material.
+This proposal includes using asciibinder and a modular approach to documentation to help us deliver high quality documentation to enable the community use AeroGear software.
+
+== Current Situation
+
+* Multiple navigation paths - there are many ways to navigate to a document, and not all give a positive experience, eg https://aerogear.org/docs/
+* Authoring requires rvm for preview, reducing contributions
+* Confusing terminology - modules, 'getting started', platforms
+* Asciidoc is used, but not a standard way
+* Stale content
+* Not much versioning - "As a developer I want to view the docs for a component I downloaded last year"
+
+== Asciibinder
+
+link:http://asciibinder.org/[Asciibinder] is used by some OSS projects for example:
+
+* https://docs.openshift.org/latest/welcome/index.html
+* https://docs.fedoraproject.org/
+* http://manageiq.org/docs/
+
+It uses asciidoc formatting, which is required for RH official documentation. Note, there is a PR to include markdown support.
+
+"AsciiBinder is for documenting versioned, interrelated projects, producing different distributions of the docs as required."
+
+== Modular documentation
+
+NOTE: This should not be confused with link:https://redhat-documentation.github.io/modular-docs/[RH Modular Docs initiative].
+
+Aerogear will consist of many components that have version specific documentation. Aerogear.org is where all that documentation will be published, however we want to make the process of contributing and publishing as easy as possible by:
+
+. Documentation for a versioned feature is authored in it's own repo. Note that this could be a code repo, or a wiki repo.
+. Documentation for all features is gathered as late as possible for publication on aerogear.org.
+
+== Demo
+
+This demo uses github and circleci, fork and build as required. 
+
+Consider the following 'feature' doc repos:
+
+* https://github.com/finp/sync-docs
+* https://github.com/finp/push-docs
+* https://github.com/finp/digger-docs
+
+Each release branch be rendered using asciibinder, for example, see link:https://5-114535426-gh.circle-artifacts.com/0/home/circleci/docs/_preview/digger/latest/installation/digger-install-intro.html[Digger docs]. 
```

Comment:
Minor grammar thing here.
'Each release branch be' => 'Each release branch will/should be'

Reply:
thanks, it was a bit early for 'write like a pirate' day